_data/capability-matrix.yml

```diff
@@ -124,9 +124,9 @@ categories:
             l2: some size restrictions in streaming
             l3: Batch mode supports a distributed implementation, but streaming mode may force some size restrictions. Neither mode is able to push lookups directly up into key-based sources.
           - class: spark
-            l1: 'Partially'
-            l2: not supported in streaming
-            l3: "Side input is actually a broadcast variable in Spark so it can't be updated during the life of a job. Spark-runner implementation of side input is more of an immutable, static, side input."
+            l1: 'Yes'
+            l2: fully supported
+            l3: A side input is actually a broadcast variable in Spark. In streaming mode, a side input could be updated between micro-batches. The distribution of side inputs to workers is not partitioned, but only a worker assigned with a relevant task will get a copy of the side input.
```

Comment:
I don't understand the here for `Flink` and `Dataflow` - clearly side inputs have size restrictions by design.. why is streaming different then batch ? and why write it here ?